docs/team/justinhoe.md

```diff
@@ -5,23 +5,48 @@ title: Justin Hoe's Project Portfolio Page
 
 ### Project: Connections
 
-Connections is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+Connections is a desktop address book application used for teaching Software Engineering principles. 
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. 
+It is written in Java, and has about 10 kLoC.
 
 Given below are my contributions to the project.
 
-* **New Feature**: 
-
+* **New Feature**: Added a Birthday Reminder List
+    * The display contacted sorted by birthdays in ascending order, starting with the 
```

Comment:
`The display contacted sorted` sounds a bit awkward?

Reply:
U are absolutely right sir